.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+| Q             | A
+| ------------- | ---
+| Bug fix?      | yes/no
+| Tests pass?   | yes                             <!-- please add some, will be required by reviewers -->
+| Fixes tickets | elifesciences/jira-import#...   <!-- #-prefixed issue number(s), if any (see: https://help.github.com/articles/autolinked-references-and-urls/#issues-and-pull-requests) -->
```

Comment:
In my experience `Fixes` is too strong for our service-based architecture as you may have 5-6 PRs that collaborate to fix a single ticket. It's more like `Helps` or `Part of`

Reply:
Given this is internal we can probably drop the `<!--` suggestions

Reply:
I've been creating checklists in issues and linked them the other way round, eg https://github.com/elifesciences/issues/issues/4354. Assumes there's an issue for a PR.

Reply:
> I've been creating checklists in issues and linked them the other way round, eg elifesciences/issues#4354. Assumes there's an issue for a PR.

I like this approach also.

Reply:
> In my experience Fixes is too strong for our service-based architecture as you may have 5-6 PRs that collaborate to fix a single ticket. It's more like Helps or Part of

I have been using `Improves: #...` but mainly to avoid the keyword `Fixes` which may trigger the issue to close. I will make a suggestion of `Issues progressed` in a separate commit.

Reply:
I've dropped the ones by the table rows